tests/services/label_data_parsing/parsing/test_resp_parsing.py

```diff
@@ -957,3 +966,339 @@ def test_video_project_classification():
+                "models": {},
+                "isVisible": True,
+                "required": 0,
+            },
+            "OBJECT_DETECTION_JOB": {
+                "content": {
+                    "categories": {"A": {"children": [], "color": "#472CED", "name": "A"}},
+                    "input": "radio",
+                },
+                "instruction": "Pose est job",
+                "mlTask": "OBJECT_DETECTION",
+                "required": 0,
+                "tools": ["pose"],
+                "isChild": False,
+            },
+            "OBJECT_RELATION_JOB": {
+                "content": {
+                    "categories": {
+                        "V": {
+                            "children": [],
+                            "color": "#5CE7B7",
+                            "name": "V",
+                            "startObjects": ["OBJECT_A"],
+                            "endObjects": ["OBJECT_B"],
+                        }
+                    },
+                    "input": "radio",
+                },
+                "instruction": "Relation job",
+                "mlTask": "OBJECT_RELATION",
+                "required": 0,
+                "isChild": False,
+            },
+            "TRANSCRIPTION_JOB": {
+                "content": {"input": "number"},
+                "instruction": "Transcription number",
+                "mlTask": "TRANSCRIPTION",
+                "required": 0,
+                "isChild": False,
+            },
+            "OBJECT_DETECTION_JOB_0": {
+                "content": {
+                    "categories": {"CAT": {"children": [], "color": "#D33BCE", "name": "Cat"}},
+                    "input": "radio",
+                },
+                "instruction": "Semantic",
+                "mlTask": "OBJECT_DETECTION",
+                "required": 0,
+                "tools": ["semantic"],
+                "isChild": False,
+                "models": {
+                    "interactive-segmentation": {"job": "OBJECT_DETECTION_JOB_0_INTERACTIVE"}
+                },
+            },
+            "OBJECT_DETECTION_JOB_0_INTERACTIVE": {
+                "content": {
+                    "categories": {"CAT": {"children": [], "color": "#D33BCE", "name": "Cat"}},
+                    "input": "radio",
+                },
+                "instruction": "Interactive Segmentation",
+                "isChild": False,
+                "isModel": True,
+                "isVisible": False,
+                "mlTask": "OBJECT_DETECTION",
+                "required": 0,
+                "tools": ["marker"],
+            },
+        }
+    }
+
+    nb_jobs = 6
+
+    json_resp = {}
+
+    project_info = Project(jsonInterface=json_interface["jobs"], inputType="IMAGE")  # type: ignore
+    parsed_jobs = ParsedJobs(project_info=project_info, json_response=json_resp)
+
+    assert len(parsed_jobs) == nb_jobs
+
+    for job_name, parsed_job_name in zip(json_interface["jobs"].keys(), parsed_jobs):
```

Comment:
since json_interface["jobs"] is a dictionary, it may be strange to assume that the order the job is kept. I would probably check: 
```
for job_name in json_interface["jobs"].keys():
    assert job_name in parse_jobs
```
unless I'm missing a use case